application/backend/app/alembic/versions/da385d690aae_schema.py

```diff
@@ -98,6 +100,9 @@ def upgrade() -> None:
         sa.ForeignKeyConstraint(["project_id"], ["projects.id"], ondelete="CASCADE"),
         sa.ForeignKeyConstraint(["training_dataset_id"], ["dataset_revisions.id"]),
         sa.PrimaryKeyConstraint("id"),
+        sa.Index("idx_model_revisions_status", "training_status"),
```

Comment:
This one is unneeded, it needs the "project_id" and "idx_model_revisions_project_status" already covers for that.